wiki/History_of_osu!/2014/id.md

```diff
@@ -1,29 +1,44 @@
 
-peppy menghapus batasan unduhan\*!
+Pada tanggal 18 Juni 2014, [peppy](https://osu.ppy.sh/users/2) mengumumkan bahwa batasan pengunduhan *beatmap* akan dihapus untuk semua *player*; namun, akan tetap ada "batasan kewarasan (sanity check)" yang akan mencegah *bot* dari pengunduhan beatmap dalam jumlah yang berlebihan \[9\].
 
-Tautan Terkait:
+## Agustus
 
-- [Download restrictions removed](https://osu.ppy.sh/home/news/2014-06-18-download-restrictions-removed)
+Pada tanggal 21 Agustus 2014, sebuah [*newspost*](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) mengumumkan bahwa akan diadakannya restrukturisasi [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team), dimana dirincikan bahwa telah dilakukannya pemecahan tim asli BAT menjadi dua tim, yaitu Beatmap Appreciation Team dan [Quality Assurance Team (QAT)](/wiki/Modding/Quality_Assurance_Team) \[10\].
 
 ## Oktober
 
-Seperti biasa, banyak hal baru ditambahkan ke osu!. Salah satunya adalah layar ketika masuk sudah diperbarui. Hal lain yang baru adalah sistem *scrolling* dalam [osu!mania](/wiki/Game_mode/osu!mania). Sistem baru ini berdasarkan pada bpm beatmap daripada *scroll* yang sudah ditetapkan! Intro dan outro baru ketika memasuki game, daripada langsung memainkan musik beatmap saat memasuki game, tema lagu yang luar biasa telah ditambahkan ke osu!. Dan kebanyakan orang mengucapkan osu! sebagai \\oss!\\, bukan \\osU!\\.
+Seperti biasa, banyak hal baru ditambahkan ke osu!. Salah satunya adalah layar ketika masuk sudah diperbarui. Hal lain yang baru adalah sistem *scrolling* dalam [osu!mania](/wiki/Game_mode/osu!mania). Sistem baru ini berdasarkan pada BPM beatmap daripada *scroll* yang sudah ditetapkan! Intro dan outro baru ketika memasuki game, daripada langsung memainkan musik beatmap saat memasuki game, tema lagu yang luar biasa telah ditambahkan ke osu!. Dan kebanyakan orang mengucapkan osu! sebagai \\oss!\\, bukan \\osU!\\.
 
-Terakhir, mengarah pada peppy membuat keputusan yang sulit untuk merilis *osu!text.exe* (sekarang *osu!cuttingedge*) dapat digunakan **siapa saja**. Hal ini memungkinkan pemain yang tidak memiliki osu!supporter untuk melakukan *debug* pada osu! di mana osu! akan diperbarui ke klien mana pun. Itu pun, jika mereka ingin menggunakan fitur eksperimental yang kurang stabil. Setiap fungsi yang ada diberikan di osu!cuttingedge **kecuali** fitur multiplayer (membutuhkan osu!supporter).
+Terakhir, mengarah pada [peppy](https://osu.ppy.sh/users/2) membuat keputusan yang sulit untuk merilis *osu!text.exe* (sekarang *osu!cuttingedge*) dapat digunakan **siapa saja**. Hal ini memungkinkan pemain yang tidak memiliki [osu!supporter](https://osu.ppy.sh/home/support) untuk melakukan *debug* pada osu! di mana osu! akan diperbarui ke klien mana pun. Itu pun, jika mereka ingin menggunakan fitur eksperimental yang kurang stabil. Setiap fungsi yang ada diberikan di osu!cuttingedge **kecuali** fitur multiplayer (membutuhkan osu!supporter).
```

Comment:
```suggestion
Di samping itu, dalam rangka meningkatkan laju pengembangan dan penumpasan bug (*debugging*) yang ada, [peppy](https://osu.ppy.sh/users/2) juga merilis klien osu! versi "osu!test" (yang kini dikenal dengan nama "Cutting Edge") secara publik di mana sebelumnya klien ini hanya tersedia bagi para pemilik [osu!supporter](https://osu.ppy.sh/home/support).
```

Reply:
same as the above, this paragraph seems to have also been drastically altered/simplified in the english article as opposed to what it previously was